dags/gcs_bucket.py

```diff
@@ -21,5 +21,6 @@
 PAX_DIR = "gs://cloud-tpu-checkpoints/pax"
 
 # GCS bucket for output
-BENCHMARK_OUTPUT_DIR = "gs://ml-auto-solutions/output/benchmark"
-XLML_OUTPUT_DIR = "gs://ml-auto-solutions/output/xlml"
+OUTPUT_DIR = "gs://ml-auto-solutions/output"
+BENCHMARK_OUTPUT_DIR = f"{OUTPUT_DIR}/benchmark"
```

Comment:
Can you add TODO comment to me to remove both BENCHMARK_OUTPUT_DIR & XLML_OUTPUT_DIR? Currently it will break if we directly remove them.